src/client/components/applayout/ContentLayout.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+class CreateMeetingLayout extends React.Component {
+  render() {
+    const customStyle = {
```

Comment:
Check out styled components for more easy styling https://www.styled-components.com/docs/basics#getting-started